src/DEFCON-1.sol

```diff
@@ -209,7 +210,9 @@ contract DssSpell {
             // This change will prevent liquidations across all collateral types
             // and is colloquially referred to as the circuit breaker.
             //
-            FlipperMomAbstract(FLIPPER_MOM).deny(registry.flip(ilks[i]));
+            if (FlipAbstract(registry.flip(ilks[i])).wards(FLIPPER_MOM) == 1) {
```

Comment:
We can remove the entire block above with enumerated ilks now.  This check should do what we want.